### PR TITLE
[Pythia8 230] MSAFETY change in ResonanceDecays

### PR DESCRIPTION
**Pull request by smuzaffar**
Decreased const MSAFETY to allow smaller mass splittings, particularly for pMSSM scans.
see https://github.com/cms-externals/pythia8/pull/19

### CODE REVIEW

**Comment by smuzaffar** [comment]
please test for CMSSW_10_2_X

**Comment by sbein** [comment]
Hi  @smuzaffar, @mrodozov, @tulamor, this should be pretty quick, but it would be great if it could be signed in the next few days. Many thanks

**Comment by mrodozov** [comment]
+externals

**Comment by mrodozov** [comment]
@sbein ow it's up to the release manager, when it is merged I'll fix cms-sw/cmsdist#5553 to include this changes

**Comment by silviodonato** [comment]
@sbein do you have a PR for master (CMSSW_11_1_X)?

**Comment by silviodonato** [comment]
+1

**Comment by silviodonato** [comment]
Let me rephrase @sbein @mrodozov I think we need https://github.com/cms-sw/cmsdist/pull/5553 also for master (CMSSW_11_1_X)

**Comment by mrodozov** [comment]
@silviodonato if we do, look at the version of pythia8 on master:
https://github.com/cms-sw/cmsdist/blob/IB/CMSSW_11_1_X/master/pythia8.spec#L1
meaning we'll need another PR for pythia8 with this 
https://github.com/cms-externals/pythia8/pull/20/files#diff-d0fa6ab377055647ab68455c68547a58L30
change, but for branch 243, correct ?

**Comment by smuzaffar** [comment]
pythia8 version 243 already contains this update https://github.com/cms-externals/pythia8/blob/cms/243/src/ResonanceDecays.cc#L30 . So this change is already in cmssw master.
